bears/css/CSSLintBear.py

```diff
@@ -17,7 +17,7 @@ class CSSLintBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
-    CAN_DETECT = {'Syntax', 'Simplification'}
+    CAN_DETECT = {'Syntax', 'Redundancy'}
```

Comment:
btw if you want to add Optimization to the set of values in coala too you'll have to do this before we merge this :D
so if we find anywhere that it needs to be optimization, youll have to add it